src/Microsoft.Health.Fhir.Core/Features/Schema/IQueryProcessor.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Core.Features.Schema
+{
+    public interface IQueryProcessor
```

Comment:
Using a Mediatr handler is the common pattern we currently have for calling functionality from the controller, (maybe `LatestSchemaVersionHandler`).